.github/workflows/ci.yml

```diff
@@ -130,39 +149,32 @@ jobs:
       - name: Install lld and jq
         run: sudo apt-get install -y lld jq protobuf-compiler
 
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+      - name: Start anvil
+        run: anvil --gas-limit 100000000000 --base-fee 1 --block-time 2 --port 3021 &
+
+      - name: Install graph CLI
+        run: curl -sSL http://cli.thegraph.com/install.sh | sudo bash
+
       - name: Build graph-node
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --bin graph-node
+          args: --bin graph-node --test integration_tests
 
-      # Integration tests are a bit flaky, running them twice increases the
-      # chances of one run succeeding.
-      - name: Run integration tests (round 1)
+      - name: Run integration tests
         id: integration-tests-1
         uses: actions-rs/cargo@v1
         env:
-          N_CONCURRENT_TESTS: "4"
-          TESTS_GANACHE_HARD_WAIT_SECONDS: "30"
-          # Reduce how long a batch will stick around to avoid timeouts
-          GRAPH_STORE_WRITE_BATCH_DURATION: 30
+          # Basically, unlimted concurrency
+          N_CONCURRENT_TESTS: "1000"
```

Comment:
Do we still need this env var then?

Reply:
I wasn't sure if we need to tweak this for GH actions, and wanted to call that out as a tunable, but if we don't need to adjust it, we can take it out.